posthog/api/proxy_record.py

```diff
@@ -58,7 +58,7 @@ def create(self, request, *args, **kwargs):
         )
 
         temporal = sync_connect()
-        inputs = CreateHostedProxyInputs(
+        inputs = CreateManagedProxyInputs(
```

Comment:
@frankh just making sure that it's ok to rename all the temporal stuff to match the new name of the feature. Hoping there's no config elsewhere that references "Hosted"

Reply:
wait no nevermind this is all in the main app